src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+
+type Props = {
+  noCompleteTodos: boolean,
+  filterBy: string,
+  setFilterBy: (value: string) => void,
+  clearCompleted: () => void,
+  leftTodos: number,
+};
+
+const navigation = ['All', 'Active', 'Completed'];
+
+export const Footer: React.FC<Props> = ({
+  noCompleteTodos,
+  filterBy,
+  setFilterBy,
+  clearCompleted,
+  leftTodos,
+}) => {
+  return (
```

Comment:
```suggestion
}) => (
```

Reply:
https://github.com/mate-academy/react_todo-app-add-and-delete/pull/327/files#r1125659726